site/sass/components/_fifty-fifty.scss

```diff
@@ -86,5 +69,17 @@
       top: -15%;
       transform: scale(0.5);
     }
+
+    &:last-of-type {
+      aspect-ratio: 1;
+      left: 50%;
+      top: 50%;
+      transform: scale(1.25) translate(25%, -50%);
```

Comment:
```suggestion
      transform: scale(1.25) translate(15%, -55%);
```

This needs to be pushed over a little at this size so that it can be read.

![image](https://user-images.githubusercontent.com/377188/160189238-e7aade94-a8dd-4e42-8ec3-3ca5d5e753a6.png)